src/renderers/shared/reconciler/ReactCompositeComponent.js

```diff
@@ -524,7 +524,7 @@ var ReactCompositeComponentMixin = {
 
     // Distinguish between a props update versus a simple state update
     if (prevParentElement === nextParentElement) {
-      nextContext = inst.context;
+      nextContext = this._processContext(nextUnmaskedContext);
```

Comment:
Looks like this is mostly an optimization so it should be ok. https://github.com/facebook/react/commit/bfcad9614ad6bebe6bbfbaa75fb121e1bad20f64#diff-c807d0f5ebbc12c44d9479dea693bef6R568

However, since you're just duplicating the line, just move it outside the block.